doc/builtin_models.rst

```diff
@@ -209,6 +209,16 @@ Exponential and Power law models
 .. autoclass:: PowerLawModel
 
 
+Two dimensional Peak-like models
+--------------------------------
+
+The one example of a two-dimensional peak is a two-dimensional Gaussian.
```

Comment:
I think this is very useful and we should probably consider extending this further to Lorentzian, Voigt, and PseudoVoigt shapes (these I like for NMR spectroscopy data) as well if it can easily be generalized. But that, of course, doesn't need to be part of this PR.

Reply:
That would indeed be good.